CHANGES.md

```diff
@@ -9,6 +9,7 @@ To be released.
 ### Backward-incompatible interface changes
 
  -  Renamed `minValue`/`maxValue` parameters to `lowerBound`/`upperBound` of `IRandom.Next()` methods.   [[#555], [#558]]
+ -  Renamed `IterateIndex()` method to `IterateIndexes` on `IStore` interface and usages. [[#462]]
```

Comment:
Could you please move the links of the issue and the PR to under the <q>Bug fixes</q> section of 0.7.0, as well?

```suggestion
 -  Renamed `IStore.IterateIndex()` method to `IterateIndexes()`.  [[#462], [#560]]
```

Reply:
Should the urls for 558 and 555 be moved up as well? (See references on Line 11)
I could do that while I am at it.

Reply:
Thank you! The links will be addressed in another PR.